0000-catalog.md

```diff
@@ -155,47 +185,55 @@ address must be constructed which identifies the storage location of the Catalog
 representation.
 
 All Grid addresses are prefixed by the 6-hex-character namespace prefix
-"621dee",  Catalogs are further prefixed under the Grid namespace with reserved
+"621dee", Catalogs are further prefixed under the Grid namespace with reserved
 enumerations of "03" ("00", "01", and "02" being reserved for other purposes)
-indicating "Grid Catalogs" and an additional org_id. Currently in Grid Product, 
-org_ids are a [6-10 digit GS1 company prefix](https://www.gs1-us.info/gs1-company-prefix/). 
-To keep the prefix length consistent, any GS1 company prefix less than 10 digits will 
-be left padded with the hexadecimal representation of the number 10 ("a") until it is 
-10 digits in length. This composite address will enable inexpensive catalog operations.
+indicating "Grid Catalogs" and an additional org_id. Currently in Grid Product,
+org_ids are a [6-10 digit GS1 company
+prefix](https://www.gs1-us.info/gs1-company-prefix/).
+To keep the prefix length consistent, any GS1 company prefix less than 10
+digits will
+be left padded with the hexadecimal representation of the number 10 ("a") until
+it is
+10 digits in length. This composite address will enable inexpensive catalog
+operations.
 
 Therefore, all addresses starting with:
 
-``` 
+```
 "621dee" + "03" + "aaaa123456" // org_id (in this case a gs1_company_prefix)
 ```
 
 are Grid GS1 Catalogs identified by the hash of the catalog name which can be
-referenced by a "catalog_product" to group products to a specific catalog. 
-
-The catalog_id format consists of 15-digit "alphanumeric string" which include 
-a fixed amount of internal "0" padding.  After the 18-hex-characters that are 
-consumed by the grid namespace prefix, the catalog, and gs1_company_prefix there 
-are 52 hex characters remaining in the address.  The 15 digits of the catalog_id can 
-be left padded with 60-hex-character zeroes and right padded with 
-2-hex-character zeroes to accommodate potential future storage associated with 
+referenced by a "catalog_product" to group products to a specific catalog.
+
+The catalog_id format consists of 70-digit "alphanumeric string" which include
+a fixed amount of internal "0" padding. After the 18-hex-characters that are
+consumed by the grid namespace prefix, the catalog, and gs1_company_prefix
```

Comment:
"the catalog" should say the "catalog prefix"

Reply:
✅